resources/windows/msi/service.wxs.erb

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+<!-- include the set of service files -->
+
+  <%- service_files.each do |service| -%>
+    <?include $(sys.SOURCEFILEDIR)include/<%=  service.path %>' ?>
```

Comment:
I don't see the value of a WIX include file, if we need to have the same information in the erb templates as well. I had thought we were making the include file a yaml file for the template parts?